compiler/luci/pass/include/luci/Pass/QuantizationParameters.h

```diff
@@ -37,6 +37,12 @@ struct LayerInfo
   QuantizationGranularity granularity;
 };
 
+enum struct QuantizationAlgorithm
+{
+  Common = 0,
```

Comment:
Q) what is the meaning of `Common` ?

Reply:
I don't know much about the math itself but it would be better to use a word that can express this.